tutorials/shaders/shader_reference/shading_language.rst

```diff
@@ -337,6 +337,23 @@ accessible outside of the shader.
 
     const float PI = 3.14159265358979323846;
 
+Constants of the ``float`` type must be initialized using ``.`` notation after the
+decimal part or by using the scientific notation. The optional ``f`` post-suffix is
+also supported.
+
+.. code-block:: glsl
+
+    float a = 1.0;
+    float b = 1.0f; // same, using suffix for clarity
+    float c = 1E-1; // gives 0.1 by using the scientific notation
+
+Constants of the ``uint`` type are using ``u`` post-suffix to differ it from the regular
+integers or by using the ``uint(x)`` built-in conversion function.
+
+.. code-block:: glsl
+
+    uint a = 1u;
+    uint b = uint(1);
```

Comment:
@Calinou, @akien-mga Maybe some errors in that paragraph or need to rephrase - Can you check?

Reply:
> By the way, does GLSL allow uppercase E in scientific notation?

yeah, as well as `F`